vignettes/getting_started.Rmd

```diff
@@ -133,6 +133,13 @@ str(gimap_dataset)
 
 The first step is running some quality checks on our data. The `run_qc()` function will create a report we can look at to assess this.
 
+The report includes several visualizations:
+
+- distribution of `count_norm` values for each sample, [TODO: description of the goals of this, check that reads are relatively uniformly represented between samples?]
```

Comment:
Hi @danieljgroso can you and Siobhan work on taking a whack at adding these TODO descriptions you've added? And then as a group we can discuss and polish them.

Reply:
sounds good!